.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
```

Comment:
Changing the action from 'freeedcom/ai-codereviewer@main' to './' suggests that the code review functionality is now being handled by a local action. Ensure that the local action is properly defined and tested to avoid any runtime issues.